integration/test_flow_integration.py

```diff
@@ -496,3 +496,23 @@ def set_moshe_time_to_none(data):
     expected = {'first_name': 'moshe', 'color': 'blue'}
     assert response.status_code == 200
     assert expected == response.output.item
+
+
+def test_cache_flushing(setup_teardown_test):
+    table = Table(setup_teardown_test, V3ioDriver(), flush_interval_secs=3)
+    controller = build_flow([
+        Source(),
+        WriteToTable(table),
+
+    ]).run()
+
+    controller.emit({'col1': 0}, 'dina', test_base_time + timedelta(minutes=25))
+    response = asyncio.run(get_kv_item(setup_teardown_test, 'dina')).output.item
+    assert response == {}
+    time.sleep(4)
```

Comment:
This could fail sometimes.

Reply:
wdym?
time.sleep may fail or the get_item?

Reply:
I mean the test may fail, because sleeping a constant time isn't guaranteed to be enough.

Reply:
flush interval is 3 secs, do you think 1 sec for doing 1 IO is not enough? i can change the numbers a bit

Reply:
Ideally the test would be deterministic, e.g. we'd wait for an event to be emitted before checking. I suppose we can leave it as is and change it if it fails.